atom/browser/autofill/atom_autofill_client.h

```diff
@@ -97,6 +97,7 @@ class AtomAutofillClient
       const base::Closure& callback) override {};
   bool ShouldShowSigninPromo() override { return false; };
   bool IsAutofillSupported() override { return true; }
+  bool AreServerCardsSupported() override { return true; }
```

Comment:
this should be false. we don't have autofill data stored in any server

Reply:
Fixed in 09f67db867a4e943a2c546dfcf273c61138f6ac4.